spec/models/pawn_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Pawn, type: :model do
+  let(:game) { FactoryGirl.create(:game) }
+  let(:user) { FactoryGirl.create(:user) }
+  let(:pawn_white) { FactoryGirl.create(:pawn, :is_on_board, color: 'white') }
+  let(:pawn_black) { FactoryGirl.create(:pawn, :is_on_board, color: 'black', row_coordinate: 1) }
+  let(:pawn_center) { FactoryGirl.create(:pawn, :is_on_board, row_coordinate: 3) }
+
+  describe 'movement of the pawn' do

```

Comment:
These look good.

1 tweak, take a look at: http://www.betterspecs.org/

Best practice is to use the describe block to name the method. So here it would be `describe '#valid_move?'`